backend/src/services/model.ts

```diff
@@ -335,6 +342,53 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   return model
 }
 
+export async function checkCollaboratorAuthorisation(
```

Comment:
This function doesn't check authorisation, could we change the name to something like `validateCollaborators`

Reply:
Also, you shouldn't need to export it as it should only be used from within this file?